cmd/template/advanced/files/workflow/github/github_action_releaser_config.yml.tmpl

```diff
@@ -1,3 +1,4 @@
+version: 2
```

Comment:
Does version 2 refer to goreleaser v2?

Reply:
Yes, without that, I noticed some errors

Reply:
In that case, goreleaser/goreleaser-action@v6 is needed in the release workflow. I didn't test v2, please check if everything is working as expected with the current .goreleaser.yml configuration. I thinkl the easiest way is to create a test repo on GitHub and push a tag.

Reply:
I tested releaser in the live repository, and it works as expected.